spec/features/viewing_properties_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "pg"
+
+feature "viewing properties" do
+  scenario "viewing properties on /spaces route" do
+    connection = PG.connect(dbname: 'makersbnb_test')
+    
+    connection.exec_params(
+      "INSERT INTO properties (property_name, property_description) VALUES($1, $2)",
+      ['A haunted house by the sea', 'It is an old haunted house, you will definitely love the fear. Enjoy the night!']
+    )
+    connection.exec_params(
+      "INSERT INTO properties (property_name, property_description) VALUES($1, $2)",
+      ['A house in the moutains', 'A big house in the mountains. You will enjoy the views and the songs of the birds.']
+    )
```

Comment:
Consider going through this process via capybara instead of directly through the database
(i.e. the steps of listing_a_property_spec first)
add property
fill in property details
hit submit
... then the rest of your test

At the moment it's testing the functionality if the database/class too much!